### ISSUE TITLE
I18N?

### ISSUE THREAD

**Issue by paolodina** [CodingForEverybody/learn-wagtail#3]
Hi and thanks for the videos. Any plan to add multi language support?

**Comment by KalobTaulien** [comment]
Yep, but unfortunately it's pretty far down the line right now. I have quite a few videos I'll be working on (that go hand-in-hand with specific commits) that the community will benefit from a bit more. The idea is to make learning Wagtail super easy, and that means going over topics people ask about most frequently (usually, some exceptions apply).

I'm going to leave this issue open until that particular video is complete though, and I've made a note that another person would like to see multi language support. 

Cheers!
